skfda/misc/_lfd.py

```diff
@@ -165,24 +176,42 @@ def __init__(self, order=None, *, weights=None, domain_range=None):
 
         self.domain_range = real_domain_range
 
-    @property
-    def order(self):
-        return len(self.weights) - 1
-
     def __repr__(self):
         """Representation of Lfd object."""
 
         bwtliststr = ""
-        for i in range(self.order + 1):
-            bwtliststr = bwtliststr + "\n" + self.weights[i].__repr__() + ","
+        for w in self.weights:
+            bwtliststr = bwtliststr + "\n" + repr(w) + ","
 
         return (f"{self.__class__.__name__}("
-                f"\nnderiv={self.order},"
                 f"\nweights=[{bwtliststr[:-1]}]"
                 f"\n)").replace('\n', '\n    ')
 
     def __eq__(self, other):
         """Equality of Lfd objects"""
-        return (self.order == other.nderic and
-                all(self.weights[i] == other.bwtlist[i]
-                    for i in range(self.order)))
+        return (self.weights == other.weights)
+
+    def constant_weights(self):
+        """
+        Return the weights of the weights if they are constant basis.
```

Comment:
What does this phrase mean?

Reply:
Probably means that I need more sleep. Thanks for noticing it.

Reply:
Fixed in 4fb0d589437d061c4ad723cfd503b9e24b663a0a.